Firestore/Firestore.podspec

```diff
@@ -22,21 +22,29 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   s.source           = { :git => 'https://github.com/TBD/Firestore.git', :tag => s.version.to_s }
   # s.social_media_url = 'https://twitter.com/<TWITTER_USERNAME>'
 
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '7.0'
+
+  s.cocoapods_version = '>= 1.4.0.beta.2'
+  s.static_framework = true
```

Comment:
Are you sure this works for swift users?

Reply:
Swift Firestore test builds and I tested Analytics and Auth quickstarts.